pybossa/forms/dynamic_forms.py

```diff
@@ -32,8 +32,7 @@ def __init__(self, *args, **kwargs):
 
     if data_access_levels:
         ProjectFormExtraInputs.amp_store = BooleanField(
-            lazy_gettext('Opt in to store annotations on Annotation Management Platform'),
-            render_kw={'checked': True})
```

Comment:
Was removing the `checked` property intentional? That is, the default for projects should be opt-in checked=True, unless the project has "L1" in input or output data, in which case the user must explicitly check the box to opt-in, if they wish to do so.

A check similar to the following?
```
render_kw={'checked': read_value_from_annotation_config['amp_store']})
```

Reply:
If the user already unchecked the opt-in and clicked 'save', when he comes back to this page, isn't it wierd to see the check box is selected again?
Maybe we should add this "default check" logic to frontend

Reply:
Isn't that what this [PR](https://github.com/bloomberg/pybossa-default-theme/pull/275/files) does? It expects the default value to already be `True` and sets it to `False` if `L1`.

I was confused why we were removing the default of `True` above, as they seems to invalidate the js from the above pr. Unless they are unrelated?

Reply:
The purpose of the PR is that IF the user changes data-classification and the new value is "L1", uncheck opt-in. 
When render the page, this checkbox should reflect the project's opt-in state instead of being always true.
`render_kw={'checked': read_value_from_annotation_config['amp_store']})` would work, will update the commit.